src/IntlDatePicker.js

```diff
@@ -0,0 +1,91 @@
+﻿require('onyx');
+
+/**
+* Contains the declaration for the {@link module:onyx/IntlDatePicker~IntlDatePicker} kind.
+* @module onyx/IntlDatePicker
+*/
+
+var
+	kind = require('enyo/kind'),
+	DatePicker = require('./DatePicker');
```

Comment:
I think we should be consistent with how we are referencing modules from the same library. In Moonstone we are using "relative" paths like this, but elsewhere in Onyx we have been using "absolute" paths i.e. `require('onyx/Picker')`.